Themes/default/languages/Login.english.php

```diff
@@ -107,6 +112,8 @@
 $txt['admin_register_group_desc'] = 'Primary membergroup new member will belong to';
 $txt['admin_register_group_none'] = '(no primary membergroup)';
 $txt['admin_register_done'] = 'Member %1$s has been registered successfully!';
+$txt['admin_register_require_agreement'] = 'Require user to accept the registration agreement';
+$txt['admin_register_require_policy'] = 'Require user to accept the privacy policy';
```

Comment:
Removing the checkboxes in `template_admin_register()` (see above) will also mean that these lines can be removed.